docs/getting-started/integrate-auth/nextjs.mdx

```diff
@@ -85,3 +94,63 @@ import VercelDeploy from '@site/src/components/VercelDeploy'
 
 <VercelDeploy />
 ```
+
+## Next.js 12 tutorial
+
+First we create a new Next.js 12 project:
+
+```shell-session
+npx create-next-app@12.1 --typescript
+cd <your-project-directory>
+```
+
+### Install Ory
+
+Ory provides integration tools that quickly combine Ory with Next.js. Let's install Ory's SDK, used to make API calls to Ory, and
+Ory's Integration Tools for JavaScript frameworks:
+
+```shell-session
+npm i --save @ory/integrations @ory/client
+```
+
+and create a special route in `[...paths].js` at `pages/api/.ory/`.
```

Comment:
Maybe say this is equivalent to the `ory tunnel` and link to the docs about it?
Also what is the path forward then from this to production? and why do we need this here instead of just e.g. adding a custom domain?